xml/update-alternatives.xml

```diff
@@ -0,0 +1,585 @@
+   300 <xref linkend="co.ua.ua-install.prio"/></screen>
+     <para>
+      The options after <option>--install</option> means:
+     </para>
+     <calloutlist>
+      <callout arearefs="co.ua.ua-install.usr-local-bin-foo">
+       <para>
+        The generic name. To avoid confusion, this is usually the script
+        name without any version numbers.
+       </para>
+      </callout>
+      <callout arearefs="co.ua.ua-install.foo">
+       <para>
+        The name of the master link. This has to be the same for both scripts.
+       </para>
+      </callout>
+      <callout arearefs="co.ua.ua-install.usr-local-bin-foo.path">
+       <para>
+        The path to the original script(s) located in
+        <filename>/usr/local/bin</filename>.
+       </para>
+      </callout>
+      <callout arearefs="co.ua.ua-install.prio">
+       <para>
+        The priority. 
+        We give <command>foo-2</command> a lower priority than
+        <command>foo-3</command>. In other words, we
+        <emphasis>prefer</emphasis> <command>foo-3</command>
+        over <command>foo-2</command>.
+        The priority can be any value you want, but it is usually a good
+        idea to leave some room inbetween. Therefor, we multiplied the
+        version by 100 which gives the priority 200 for <command>foo-2</command>
+        and 300 for <command>foo-2</command>.
+       </para>
+      </callout>
+     </calloutlist>
+    </step>
+    <step>
+     <para>
+      Check the master link:
+     </para>
+     <screen>&prompt.root;<command>update-alternatives</command> --display foo
+foo - auto mode
+  link best version is /usr/local/bin/foo-3
+  link currently points to /usr/local/bin/foo-3
+  link foo is /usr/local/bin/foo
+/usr/local/bin/foo-2 - priority 200
+/usr/local/bin/foo-3 - priority 300</screen>
+    </step>
+   </procedure>
+   <para>
+    After you have executed the above steps, you can use the master link
+    <command>/usr/local/bin/foo</command>.
+   </para>
+   <para>
+    If needed, you can install additional alternatives. In case you want
+    to remove one alternatives, use this command:
+   </para>
+   <screen>&prompt.root;<command>update-alternative</command> --remove foo /usr/local/bin/foo-2</screen>
+   <para>After this script has been removed, the &as; for the foo group looks
+    like this:</para>
+   <screen>&prompt.root;<command>update-alternative</command> --display foo
+foo - auto mode
+  link best version is /usr/local/bin/foo-3
+  link currently points to /usr/local/bin/foo-3
+  link foo is /usr/local/bin/foo
+/usr/local/bin/foo-3 - priority 300</screen>
+  </sect1>
+
+  <sect1 xml:id="sec.ua.slaves">
+   <title>Defining Dependent Alternatives</title>
+   <para>
+    If you have alternatives, the script itself is not enough. Usually
+    scripts comes with a manpage or some other configuration options.
+    In case you need to create alternatives which are dependent from a master
```

Comment:
*In case you need to create* -> To create

Reply:
done